aospy/calc.py

```diff
@@ -650,9 +660,36 @@ def region_calcs(self, arr, func, n=0):
             else:
                 method = getattr(reg, func)
                 data_out = method(arr)
-            reg_dat.update({reg.name: data_out})
+                if bool_pfull:
+                    # Don't apply e.g. standard deviation to coordinates.
+                    if func not in ['av', 'ts']:
+                        method = reg.ts
+                    # Convert Pa to hPa
+                    coord = method(pfull) * 1e-2
```

Comment:
Yes, this was laziness on my part.  The warning message about the conversion was distracting to me, so I just did this.  I will tweak `to_hpa` for better control of the warnings.